Add ep 8 trailer full (#59)
psf/the-invisibles: content/english/episodes/ep-8.md

---
title: "Episode 8: with Mojdeh Rastgoo"
meta_title: "Episode 8 with Mojdeh Rastgoo"
description: "Learn about Mojdeh's journey. Since discovering Python and open source community in 2016, Mojdeh has been actively contributing to the community and striving for diversity and inclusion by pariticapting in DEI, CoC communities and co-organzing meetups"
date: 2025-04-15T05:00:00-07:00
host: ["Tereza", "Georgi"]
draft: false
tags: ["python", "iran", "france", "pyladies", "psf-coc-wg", "pypodcats", "community", "project manager"]
publish_date: 2025-04-15T00:00:00-07:00
podcast_file: "/audio/hidden-figures-ep8.mp3"
podcast_duration: 1647.6
episode_image: "images/ep8-mojdeh.png"
podcast_bytes: "66615526" # the length of the episode in bytes
images: ["images/ep8-mojdeh.png", "images/mojdeh.jpg"]
explicit: false 
type: 'episode'
episode: '8'
season: 1
transcript: '/transcript/hidden-figures-of-python-ep8.srt'
episode_type: full
layout: episode
youtube_id: "Gh1OOv2qtTA"
  
speaker:
  url: "/speakers/mojdeh-rastgoo/"
  title: "Mojdeh Rastgoo"
  image: "/images/mojdeh.jpg"
  description: "

> Mojdeh Rastgoo is a Machine Learning Data Scientist and Project Manager based in Pris.

Reviewer: ```suggestion
Mojdeh Rastgoo is a Machine Learning Data Scientist and Project Manager based in Paris.
```

> She has a PhD in computer sicence and has been an active memnber of the community since 2018. 

Reviewer: ```suggestion
She has a PhD in Computer Science and has been an active member of the community since 2018. 
```

> She is a co organizer of PyLadies Paris and member of PSF Code of Conduct Working-Group since mid 2023.

Reviewer: ```suggestion
She is a co-organizer of PyLadies Paris and member of PSF Code of Conduct Working Group since mid 2023.
```

And recently she has joined the PyPodcats team as the fifth organizer.  

> She participated in organization of EuroSciPy 2023/2024 conference, JupyterCon 2023, PyCon.DE & PyData Berlin 2024, and PyLadies Paris meetups since 2022. 

Reviewer: ```suggestion
She participated in organization of EuroSciPy 2023/2024 conference, JupyterCon 2023, PyCon DE & PyData Berlin 2024, and PyLadies Paris meetups since 2022. 
```

> She has also given introductory python tutorials in several occaisions.

Reviewer: ```suggestion
She has also given introductory Python tutorials in several occasions.
```

"
  social:
  - name: github
    icon: fa-brands fa-github
    link: https://github.com/mrastgoo

  - name: twitter
    icon: fa-brands fa-x-twitter
    link: https://x.com/RastgooMojdeh

  - name: linkedin
    icon: fa-brands fa-linkedin
    link: https://www.linkedin.com/in/mojdeh-rastgoo-6462753a/

  - name: web
    icon: fa-solid fa-globe
    link: https://mrastgoo.github.io./
  
---

> We interviewed Mojdeh Rastgoo the new memner of PyPodcats.

Reviewer: ```suggestion
We interviewed Mojdeh Rastgoo the new member of PyPodcats.
```

> Since discovering Python and open source communities, Mojdeh has been actively involved in the python community, since she gave her first tutorial in 2018.

Reviewer: ```suggestion
Since discovering Python and open source communities, Mojdeh has been actively involved in the Python community. She gave her first tutorial in 2018 at
```

> She is a member of PSF CoC working group, co organizer of PyLadies Paris and now PyPodcast host!!

Reviewer: ```suggestion
She is a member of PSF CoC Working Group, co-organizer of PyLadies Paris and now PyPodcast host!!
```

In this chat, Mojdeh tells us more about herself and her passion about the community.

> In addition we give a short recap of 2024, and talk about our plans for 2025. 

Reviewer: ```suggestion
In addition, we give a short recap of 2024, and talk about our plans for 2025. 
```

> We also introduce couple of new changes in our organization, such as open-collective account, wehere you can support us.

Reviewer: ```suggestion
We also introduce couple of new changes in our organization, such as an Open Collective account, where you can support us.
```

> Be sure to listen to the episode to hear about our plans and get to know your new host Mojdeh !

Reviewer: ```suggestion
Be sure to listen to the episode to hear about our plans and get to know your new host Mojdeh!
```

## Topic discussed

- Introductions
- What are we up to nowadays 
- Getting to know Mojdeh 
- What Saint-Gobain does and Mojdeh roles in Saint-Gobain
- How did she ended up in Europe 
- Her involvement in the community
- PyLadies Paris
- Public speaking and Lightning talks
- Rols of LLM in today's life 
- Personal projects 
- Growth mindset and changing perspectives
- Personal interests and hobbies 
- Podcats reflection and plans for the future
- Founding and supports

## Links from the show

- https://pypodcats.live/blog/kickstart_2025/
- https://opencollective.com/pypodcats

> 

Reviewer: Maybe add a few more links:
- Erasmus Mundus scholarship
- links to episodes of Vicky, Joanna, and Jessica (because they were mentioned during the session)
- Think Again book
- PyPodcats Open Collective

draft: false
tags: ["python", "iran", "france", "pyladies", "psf-coc-wg", "pypodcats", "community", "project manager"]
publish_date: 2025-04-15T00:00:00-07:00
podcast_file: "/audio/hidden-figures-ep8.mp3"
podcast_duration: 1647.6
episode_image: "images/ep8-mojdeh.png"
podcast_bytes: "66615526" # the length of the episode in bytes
images: ["images/ep8-mojdeh.png", "images/mojdeh.jpg"]
explicit: false 
type: 'episode'
episode: '8'
season: 1
transcript: '/transcript/hidden-figures-of-python-ep8.srt'
episode_type: full
layout: episode
youtube_id: "Gh1OOv2qtTA"
  
speaker:
  url: "/speakers/mojdeh-rastgoo/"
  title: "Mojdeh Rastgoo"
  image: "/images/mojdeh.jpg"
  description: "
Mojdeh Rastgoo is a Machine Learning Data Scientist and Project Manager based in Paris.
She has a PhD in computer sicence and has been an active memnber of the community since 2018. 
She is a co-organizer of PyLadies Paris and member of PSF Code of Conduct Working Group since mid 2023.
And recently she has joined the PyPodcats team as the fifth organizer.  

She participated in organization of EuroSciPy 2023/2024 conference, JupyterCon 2023, PyCon DE & PyData Berlin 2024, and PyLadies Paris meetups since 2022.

> She has also given introductory python tutorials in several occasions.

Reviewer: ```suggestion
She has also given introductory Python tutorials in several occasions.
```

    link: https://mrastgoo.github.io./
  
---

We interviewed Mojdeh Rastgoo, the newest member of PyPodcats!

Since discovering Python and the open-source community, Mojdeh has been actively involved in the Python ecosystem. She gave her first tutorial in 2018 at EuroSciPy and has since contributed in many ways. She is a member of the PSF Code of Conduct Working Group, a co-organizer of PyLadies Paris, and now a host of PyPodcats!

In this episode, Mojdeh shares more about herself and her passion for the community. We also take a look back at 2024, discuss our plans for 2025, and introduce a few new changes, including our Open Collective account, where you can support us.

Be sure to listen to the episode to hear about our plans and get to know your new host Mojdeh!

## Topic discussed

- Introductions
- What are we up to nowadays 
- Getting to know Mojdeh 
- What Saint-Gobain does and Mojdeh roles in Saint-Gobain
- How did she ended up in Europe 
- Her involvement in the community
- PyLadies Paris
- Public speaking and Lightning talks
- Rols of LLM in today's life 
- Personal projects 
- Growth mindset and changing perspectives
- Personal interests and hobbies 

> - Podcats reflection and plans for the future

Reviewer: Should this say PyPodcats or Podcast instead of Podcats?

Author: PyPodcats probably, sorry for the mistake, I will correct it

- Founding and supports

## Links from the show

- [PyPodcats Kickstart 2025](https://pypodcats.live/blog/kickstart_2025/)
- [PyPodcats OpenCollective](https://opencollective.com/pypodcats)
- [Erasmus+ - Erasmus Mundus Master Program](https://erasmus-plus.ec.europa.eu/opportunities/opportunities-for-individuals/students/erasmus-mundus-joint-masters)
- [Think again by Adam Grant](https://adamgrant.net/book/think-again/)
- [PyPodcats episode 1 with Joanna Jablonski](https://pypodcats.live/episodes/ep-1/)
- [PyPodcats episode 3 with Vicky Twomey-Lee](https://pypodcats.live/episodes/ep-3/)
- [PyPodcats episode 4 with Jessica Tegner](https://pypodcats.live/episodes/ep-4/)

## Links for PyLadies Paris 

> - [PyLadies paris meetup](https://www.meetup.com/fr-FR/pyladiesparis/)

Reviewer: ```suggestion
- [PyLadies Paris meetup](https://www.meetup.com/fr-FR/pyladiesparis/)
```
